sormas-backend/src/main/java/de/symeda/sormas/backend/caze/caseimport/CaseImportFacadeEjb.java

```diff
@@ -351,7 +351,7 @@
 		List<String> invalidColumns = new ArrayList<>();
 
 		for (int i = 0; i < values.length; i++) {
-			String value = values[i];
+			String value = values[i] != null ? values[i].trim() : values[i];
```

Comment:
Just a suggestion. `StringUtils.trimToNull(values[i])` achieves the same and maybe it's easier to read.